New Information Architecture

nearestnabors opened the pull request:
* Moves docs to new IA structure
* Removes top nav
* Updates internal links to new URLs
* Adds permanent redirects to new URLs

[Preview](https://docs-git-new-ia-backup-arcade-ai.vercel.app/en/home)

<img width="1247" height="847" alt="Screenshot 2025-12-30 at 10 59 04 PM" src="https://github.com/user-attachments/assets/c1bfdac5-956f-4701-b7b8-1e606282ec91" />



<!-- CURSOR_SUMMARY -->
---

> [!NOTE]
> Major documentation IA overhaul with new top-level sections and navigation.
> 
> - Introduces new sections and metadata (`get-started`, `guides`, `references`, `resources`) with many new/relocated pages and `_meta.tsx` files; removes legacy `home` structure
> - Updates internal links across guides to new paths (e.g., `/home/...` → `/guides/...`, `/references/...`, `/resources/integrations/...`)
> - Refactors site nav metadata in `app/en/_meta.tsx` (iconized titles, breadcrumb/layout settings)
> - Minor UI/content tweaks: comment out `Badge` in `Logo`, adjust `editPage` copy in `_dictionaries/en.ts`, convert several TS `interface` types to `type`
> - Script maintenance: `.github/scripts/update-example-dates.js` now uses `node:` imports and points to `app/en/resources/examples/page.mdx`
> - CI/workflows: minor copy/whitespace updates in `translate-docs.yml` (example `single_file` path)
> - Repo hygiene: add `*.bak` to `.gitignore`
> 
> <sup>Written by [Cursor Bugbot](https://cursor.com/dashboard?tab=bugbot) for commit f41d1bf48fe87e41322d0ac0b140909201f48a2d. This will update automatically on new commits. Configure [here](https://cursor.com/dashboard?tab=bugbot).</sup>
<!-- /CURSOR_SUMMARY -->

evantahler commented:
One more nit - the `get-started/quickstarts/call-tool-agent` page doesn't actually teach you how to use Arcade in an agent - it teaches you how to use Arcade's tool programmatically, like any old function.  I think either the title is misleading, or the tutorial should end with the use of the smallest agent framework available (e.g. OpenAI AgentsSDK?)

nearestnabors commented:
@evantahler

> Navigationally, the list of integrations/toolkits moving under 'resources' is fine, but this page is really important for the "should I use Arcade" decision making process. I think that means is should have a top-nav link as well. It should also be called out in some way to differentiate the importance of that page.

I also feel uneasy about this. But here's what we noticed in user testing—no one was using the Global Nav Bar. They would go to arcade.dev and click the Integrations link from there. When we added the link to the sidebar, they started clicking from there. So this version removes the global navigation bar entirely. Would you feel better if we added it back and had duplication?

> I agree with @EricGustin that the tool-type categorization...We should focus on the starter tools and de-emphasize the starter tools - they are mixed now.

Is this a filter we can add to the main page? Because that seems to be the way people prefer to navigate the tools—by filters.

> The new 'about arcade' is good! How does that relate to the 'home' page - can they be merged or are they doing different jobs?

The home page will need to be retooled, IMO ideally to shunt people to different resources, like react.dev and docs.agentql.com do. (Generally, docs landing pages are not the place to sell to new users, but a place to triage incoming users new and old to the right place to get their answers!) I agree we need to update it. But it's for a different PR (and possible workshopping!)

nearestnabors commented:
They're already in docs triage with follow up milestones @evantahler  :) However, I did have to push some fixes to broken links and redirects. Please hold
